modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/internal/freemarker/tool/FreeMarkerTool.java

```diff
@@ -1128,11 +1186,7 @@ public boolean isGeneratePermissions(
 		Map<String, Schema> propertySchemas = schema.getPropertySchemas();
 
 		if (MapUtil.isEmpty(propertySchemas) ||
-			!propertySchemas.containsKey("permissions") ||
-			!containsJavaMethodSignature(
-				javaMethodSignatures, "get" + schemaName + "PermissionsPage") ||
-			!containsJavaMethodSignature(
-				javaMethodSignatures, "put" + schemaName + "PermissionsPage")) {
+			!propertySchemas.containsKey("permissions")) {
```

Comment:
Not sure if removing the old code here was more clear than now. Let's disccuss after fixing all the SF issues in the ftl